tests/cats/test_cat_command.py

```diff
@@ -42,6 +43,7 @@ async def test_cat_mint(
     await full_node_api.wait_for_wallet_synced(wallet_node=wallet_node_0, timeout=20)
 
     self_address = encode_puzzle_hash(await wallet_0.get_new_puzzlehash(), "xch")
+    assert isinstance(wallet_0.wallet_state_manager.private_key, PrivateKey)
```

Comment:
`WalletStateManager` has a method called `get_master_private_key()` which will throw a more helpful error than this.